test-data/dpkg/emacs-rdepends.dpkg

```diff
@@ -0,0 +1,171 @@
+emacs25
+Reverse Depends:
+ |auctex
+    emacs25-lucid
+    emacs25-nox
+  elpa-ztree
+    emacs25-lucid
+    emacs25-nox
+ |yatex
+    emacs25-lucid
+    emacs25-nox
+  elpa-yasnippet
+    emacs25-lucid
+    emacs25-nox
+ |w3m-el-snapshot
+    emacs25-lucid
+    emacs25-nox
+ |w3m-el
+    emacs25-lucid
+    emacs25-nox
+  twittering-mode
+    emacs25-lucid
+    emacs25-nox
+ |speechd-el
+    emacs25-lucid
+    emacs25-nox
+ |slime
+    emacs25-lucid
+    emacs25-nox
+ |sepia
+    emacs25-lucid
+    emacs25-nox
+ |search-citeseer
+    emacs25-lucid
+    emacs25-nox
+  elpa-recursive-narrow
+    emacs25-lucid
+    emacs25-nox
+ |psgml
+    emacs25-lucid
+    emacs25-nox
+  proofgeneral
+    emacs25-lucid
+    emacs25-nox
+ |org-mode
+    emacs25-lucid
+    emacs25-nox
+  elpa-muse
+    emacs25-lucid
+    emacs25-nox
+  elpa-monokai-theme
+    emacs25-lucid
+    emacs25-nox
+ |minlog
+    emacs25-lucid
+    emacs25-nox
+ |mhc
+    emacs25-lucid
+    emacs25-nox
+ |mh-e
+    emacs25-lucid
+    emacs25-nox
+ |mew-beta
+    emacs25-lucid
+    emacs25-nox
+ |mew
+    emacs25-lucid
+    emacs25-nox
+  elpa-makey
+    emacs25-lucid
+    emacs25-nox
+ |lookup-el
+    emacs25-lucid
+    emacs25-nox
+ |ilisp
+    emacs25-lucid
+    emacs25-nox
+  elpa-iedit
+    emacs25-lucid
+    emacs25-nox
+  elpa-ido-vertical-mode
+    emacs25-lucid
+    emacs25-nox
+  elpa-ido-ubiquitous
+    emacs25-lucid
+    emacs25-nox
+  elpa-ido-completing-read+
+    emacs25-lucid
+    emacs25-nox
+  elpa-hydra
+    emacs25-lucid
+    emacs25-nox
+  elpa-helm-core
+    emacs25-lucid
+    emacs25-nox
+  elpa-helm
+    emacs25-lucid
+    emacs25-nox
+ |haskell-mode
+    emacs25-lucid
+    emacs25-nox
+ |goby
+    emacs25-lucid
+    emacs25-nox
+  elpa-git-timemachine
+    emacs25-lucid
+    emacs25-nox
+  elpa-fsm
+    emacs25-lucid
+    emacs25-nox
+  elpa-expand-region
+    emacs25-lucid
+    emacs25-nox
+  emacs25-nox
+    emacs25-lucid
+    emacs25-nox
+  emacs25-nox
+    emacs25-lucid
+  emacs25-lucid
+    emacs25-lucid
+    emacs25-nox
+  emacs25-lucid
+    emacs25-nox
+ |emacs-window-layout
+    emacs25-lucid
+    emacs25-nox
+  elpa-jabber
+    emacs25-lucid
+    emacs25-nox
+  elpa-deferred
+    emacs25-lucid
+    emacs25-nox
+  elpa-concurrent
+    emacs25-lucid
+    emacs25-nox
+ |emacs-calfw
+    emacs25-lucid
+    emacs25-nox
+  elpa-anzu
+    emacs25-lucid
+    emacs25-nox
+ |egg
+    emacs25-lucid
+    emacs25-nox
+ |e2wm
+    emacs25-lucid
+    emacs25-nox
+  elpa-discover-my-major
+    emacs25-lucid
+    emacs25-nox
+ |dh-elpa
+    emacs25-lucid
+    emacs25-nox
+ |ddskk
+    emacs25-lucid
+    emacs25-nox
+ |ddskk
+    emacs25-lucid
+    emacs25-nox
+  elpa-clues-theme
+    emacs25-lucid
+    emacs25-nox
+  elpa-avy
+    emacs25-lucid
+    emacs25-nox
+  auto-install-el
+    emacs25-lucid
+    emacs25-nox
+ |auto-complete-el
+    emacs25-lucid
+    emacs25-nox
```

Comment:
Wait, this output does not ring a bell... It looks wildly different from `emacs-depends`.  Can you test the "Show (r)deps" actions for dpkg?  We might need to use a different rdep command.

Reply:
@DamienCassou Did you have time to try this out?